src/os.h

```diff
@@ -25,6 +25,48 @@ enum ThreadState {
     THREAD_SLEEPING
 };
 
+struct ProcessInfo {
+    int _pid;
+    int _ppid;
+    char _name[16];              // Process name from /proc/{pid}/stats
+    char _cmdline[2048];         // Command line from /proc/{pid}/cmdline
+    unsigned int _uid;           // User ID
+    unsigned char _state;        // Process state (R, S, D, Z, T, etc.)
+    unsigned long _start_time;   // Process start time (clock ticks since boot)
```

Comment:
Is `unsigned long` a proper type for all these values? Keep in mind that `unsigned long` is 32-bit on 32-bit systems.